snapcraft.yaml

```diff
@@ -0,0 +1,20 @@
+name: drive
+summary: Google Drive client for the commandline
+version: current
```

Comment:
Hi!
Great work!
Just a suggestion. The version field should be a string, so you should put it in quotes.
Also, if you're tracking the master branch in github, it might be better to set:
`version: 'master'`
What do you think?